### PR TITLE
Add level 15 - GPS based level

### PR DESCRIPTION
**Pull request by Uriopass**
You have to type the coordinates in a GPS lookup, and it opints to the hollywood panels (letters).
If you reverse latitude and longitude, it's in the antarctica ocean.

### CODE REVIEW

**Comment by sharkdp** [comment]
Love the idea! Thanks!

Experimental version is up here: https://shark.fish/curiosity/experimental/level015/cinema.html

Unfortunately, I saw the title and message of the pull request and could not try to solve it without knowing the solution... so it's hard to judge how difficult this really is. But I guess there should be a slight hint as to what these two numbers represent, don't you think? I could think of two possible versions:
- change `cinema.html` to something more Lat/Lon specific.. like `compass.html` or similar.
- Use a different coordinate representation (like `34°8'2''`), possibly in some obfuscated form

**Comment by Uriopass** [comment]
Compass is great ! I thought about names like GPS.html or map.html but theses were too obvious.

File: levels/level014/cartel.html [hunk 1]
```diff
@@ -1 +1 @@
-Excellent! <a href="../finish/eof.html">Go to the next level</a>.
+Excellent! <a href="../level5/cinema.html">Go to the next level</a>.
```
Comment (sharkdp) [inline levels/level014/cartel.html]: This should be `../level015/cinema.html`, right?

Comment (Uriopass) [reply levels/level014/cartel.html]: Oh you're right, my bad, I'll commit the changes to compass and the 015

**Comment by sharkdp** [comment]
Thank you for the updates! I'll probably also move this to a lower level number, once I have #13 figured out.

**Comment by sharkdp** [comment]
This is now live at: https://shark.fish/curiosity/level015/compass.html

Thanks again!

**Pull request merged**